### DISCUSSION TITLE
withoutGlobalScopes on BelongsToMany relation

### DISCUSSION THREAD

**Discussion by shadowbane** [laravel/framework#50956] [Q&A]
Laravel Version: 11.2.0
<img width="365" alt="Screenshot 2024-04-08 at 14 35 37" src="https://github.com/laravel/framework/assets/6396154/29b9a5b0-ca3c-4231-bb15-093e5486f20d">

Hi, I've been using query scope and sometimes disable it like the screenshot when loading the relations. But somehow, in Laravel 11 it's not working anymore. It was working before on Laravel 10.

Is there any changes that I should notice, regarding to query scopes on Laravel 11?

Thanks for the help :)

**Comment by tfortin** [comment]
Hello,

Same issue here. It seems that changes on [BelongsToMany relationship](https://github.com/laravel/framework/compare/10.x...11.x#diff-71a75a222da08e3c0c09631652c1459ff44c3e25d84c05ce48877c0a68fd20cdR1566) have made it loading scopes before we could tell it not to use them. And calling `->withoutGlobalScopes()` before `->belongsToMany()` cannot be done.

Not sure what to do with that.

**Reply by tfortin** [reply]
I'm answering my own post, but it seems that a PR has already been made! https://github.com/laravel/framework/pull/50953
